code/modules/awaymissions/mission_code/ruins/watcher_grave.dm

```diff
@@ -0,0 +1,256 @@
+/obj/effect/mob_spawn/corpse/goliath/pierced
+	corpse_description = "Seems to have been pierced through the heart by a Watcher spike."
+	naive_corpse_description = "It's got a pretty big boo-boo, might need one of the large plasters."
+
+/obj/effect/mob_spawn/corpse/watcher/goliath_chewed
+	corpse_description = "Prior to its death, it was badly mangled by the jaws of a Goliath."
+	naive_corpse_description = "It's all tuckered out after playing rough with a Goliath."
+
+/obj/effect/mob_spawn/corpse/watcher/crushed
+	corpse_description = "Crushed by a rockslide, it seemed to have been scraping frantically at the rocks even as it perished."
+	naive_corpse_description = "All of those rocks probably don't make a comfortable blanket."
+
+
+#define WATCHER_EGG_LIVELY_MOD 0.75
+#define WATCHER_EGG_ACTIVE_MOD 0.5
```

Comment:
Comments for these like this, would be great. (these are based on your prev comment on github)
```suggestion
/// if greater than lively mod, jiggles faster
#define WATCHER_EGG_LIVELY_MOD 0.75
/// If greater than active mod, egg is active and jiggles
#define WATCHER_EGG_ACTIVE_MOD 0.5
```